LobbyManager.h

```diff
@@ -0,0 +1,60 @@
+#ifndef TICTACTOE_SERVER_LOBBYMANAGER_H
+#define TICTACTOE_SERVER_LOBBYMANAGER_H
+
+#include "SearchManager.h"
+#include <unordered_map>
+#include <random>
+
+class LobbyManager {
+public:
+	struct Lobby {
+		SearchManager::PlayerInfo player1;
+		SearchManager::PlayerInfo player2;
+		bool player1_active;
+		bool player2_active;
+		unsigned search_id;
+
+		Lobby(SearchManager::PlayerInfo player1, SearchManager::PlayerInfo player2, unsigned id) : player1(player1), player2(player2), search_id(id){}
+
+	};
+
+private:
+	unordered_map<unsigned, Lobby> active_lobbies;
```

Comment:
This is a compile-time error. There is no class/struct/union named "unordered_map" in the current namespace. Use `std::unordered_map`